de-lib/src/main/java/org/iplantc/de/apps/client/AppDetailsView.java

```diff
@@ -32,24 +32,29 @@
     @JsType
     interface Presenter {
 
+        @SuppressWarnings("unusable-by-js")
         void go(App app,
                 String searchRegexPattern);
```

Comment:
I wouldn't think these `Presenter.go` methods would be used by our javascript code 🤔 
Should these `go` methods be annotated with `@JsIgnore` instead?

Reply:
Oh whoops 😊 you're probably right